problems/removeEvenStrings.js

```diff
@@ -6,8 +6,11 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
-
+function removeEvenStrings(arr) {
+    let oddStrings = arr.filter((e, i) => {
+        return arr[i].length % 2 !== 0
```

Comment:
Prefer positive when possible === 1 instead of !== 0 makes code slightly more readable.